Rozdział 8/Hangman2.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hangman2!</title>
+</head>
+<body>
+    <script>
+    
+    var words = ["javascript", "monkey", "amazing", "pancake", "tractor", "blablabla"];
+
+    
+    var pickWord = function() {
+        return words[Math.floor(Math.random() * words.length)];
+    };
+
+    var setupAnswerArray = function (word) {
+        var answerArray = [];
+    for (var i = 0; i < word.length; i++) {
+        answerArray[i] = "_";
+        }
+        return answerArray;
+    }
+    
+    var word = pickWord();
+    var answerArray = setupAnswerArray(word);
+    var remainingLetters = word.length;
+
+    
+
+    var showPlayerProgress = function (answerArray) {
+        alert(answerArray.join(" "));
+    }
+
+    var getGuess = function () {
+        return prompt("Guess a letter, or click Cancel to stop playing.").toLowerCase();
```

Comment:
jak wynik prompt będzie nullem (user kliknie cancel) to gierka Ci się położy na plecy , bo nie będzie mogła wuwołać toLowerCase na nullu

Mógłbyś zrobić coś w stylu:

```
var getGuess = function () {
  var promptResult = prompt('Guess a letter, or click Cancel to stop playing.');

  return promptResult ? promptResult.toLowerCase() : promptResult;
}
```

nie wiem czy ternary operator kojarzysz, jak nie to łap https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator